config/stack_ncar.yaml

```diff
@@ -261,3 +261,7 @@ atlas:
   build: YES
   version: release-stable
   repo: jcsda
+
+gsl:
+  build: YES
+  version: 2.6
```

Comment:
same comment as above.
please move under gsl-lite section

Reply:
This file no longer exists in develop.  It was removed from my branch as well once I pulled in changes from develop.